src/test/java/com/wikia/webdriver/testcases/fandom/SearchInputTest.java

```diff
@@ -0,0 +1,30 @@
+package com.wikia.webdriver.testcases.fandom;
+
+import com.gargoylesoftware.htmlunit.javascript.host.URL;
+import com.wikia.webdriver.common.core.Assertion;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.Assertion'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)